pkg/kubelet/dockertools/docker.go

```diff
@@ -316,7 +316,11 @@ func (d *dockerContainerCommandRunner) PortForward(podInfraContainerID string, p
 	}
 
 	containerPid := container.State.Pid
-	// TODO use exec.LookPath for socat / what if the host doesn't have it???
+	// TODO what if the host doesn't have it???
+	path, lookupErr := exec.LookPath("socat")
+	if lookupErr != nil {
+		return fmt.Errorf("The socat command cannot be find in PATH=%s.", path)
```

Comment:
Agree with the PATH information not necessarily useful.

Regarding not mentioning socat: That would be very obscure. Not being able to know the cause of the  "Port forwarding is unavailable" is very frustrating. Why hiding it ?